training/hc-leipzig-7t-mp2rage/nnUNetv2_mp2rage_random_train_val_split_to_json.py

```diff
@@ -0,0 +1,70 @@
+import pandas as pd
+import numpy as np
+from sklearn.model_selection import KFold
+import json
+
+# Load the data from the CSV file
+train_test_dataframe = pd.read_csv('hc-leipzig-7t-mp2rage_train-test_split.csv')
+
+# Get the training data only
+train_dataframe = train_test_dataframe[train_test_dataframe['Train/Test'] == 'Train']
+
+# Extract unique subjects and rename them to SCDATA_<number>
+subjects = train_dataframe['Subject'].unique()[:]
+
+# Get only number from the subject name
+subjects = [int(subj.split('-')[1]) for subj in subjects]
+
+# Rename the subjects
+renamed_subjects = [f"SCDATA_{subj}" for subj in subjects]
```

Comment:
do we need to do this renaming? by doing it, the output `csv` file contains `sub-` but the output `json` file contains `SCDATA`. this is confusing. I would stick with the original filenames: `sub-`